src/pages/Settings/sections/MyPostsSettings.css

```
@@ -244,3 +244,285 @@
+  inset: 0;
+  z-index: 1000;
+  background: rgba(0, 0, 0, 0.6);
+  backdrop-filter: blur(4px);
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: 24px;
+  animation: mps-fadeIn 0.2s ease;
+}
+
+@keyframes mps-fadeIn {
```

Stylelint (17.6.0)

[error] Expected keyframe name "mps-fadeIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)